hazelcast-tpc-engine/src/main/java/com/hazelcast/internal/tpc/Eventloop.java

```diff
@@ -58,7 +58,7 @@
  *     <li>Tasks from some asynchronous eventing system that interacts with I/O. </li>>
  * </ol>
  */
-@SuppressWarnings({"checkstyle:VisibilityModifier", "rawtypes"})
+@SuppressWarnings({"checkstyle:VisibilityModifier", "checkstyle:declarationorder", "rawtypes"})
```

Comment:
Just a question: Are we going to get rid of (maybe some?) warnings before merging to master, or do we plan to leave them with these suppressions?

Reply:
With suppression. This is a nonsense checkstyle constraint on code. You should never group based on the access modifier but on what belongs to each other.

Reply:
This whole section needs to be improved anyway due to potential false sharing issues. THe (mutable) state that is accessed by the eventloop thread should not be on the same cache lines as state accessed by other threads. So I need to add padding.